app/src/etc/supervisord.conf

```diff
@@ -46,7 +46,17 @@ command = node NodeJSServer.js
 directory=/home/bdt/map/src/map_nodejs_server/
 stdout_logfile=/var/log/NodeJSServer.log
 stderr_logfile=/var/log/NodeJSServer.log
-autostart=false                ; start at supervisord start (default: true)
+autostart=true                 ; start at supervisord start (default: true)
 autorestart=false              ; restart at unexpected quit (default: true)
 stopasgroup=true               ; stop all subprocesses together
 killasgroup=true               ; kill all subprocesses together
+
+[program:startup]
+command = /bin/bash /home/bdt/map/src/map_scripts/startup_after_shutdown.sh
```

Comment:
I'm sorry I don't understand. What do you mean by included? Add a link to the bdt PR in the conf file?